PREREQUISITES.md

```diff
@@ -51,14 +51,42 @@ openssl genrsa -3 -out private_rsa_key.pem 3072
 export PDO_ENCLAVE_PEM=`pwd`/private_rsa_key.pem
```

Comment:
Please, add PKG_CONFIG_PATH to the Environmental Variables list

Reply:
We already reference the environment file in SGX which sets the PKG_CONFIG_PATH variable. I would suggest that we simply add the documentation around that reference.

Reply:
Package config path is not required if you install the OpenSSL packages for your distribution. I added a bunch of text in the OpenSSL section to indicate this was the recommended path. Installing these packages directly means they will be picked up correctly from the system - no PKG_CONFIG_PATH required.